spec/configuration_spec.rb

```diff
@@ -396,4 +396,15 @@ def debug(name, &block)
       expect(second_array).to eq([1, 2])
     end
   end
+
+  describe "vendor_path" do
+    it "should have the default vendor path" do
```

Comment:
I prefer to avoid "should" in doc strings (https://github.com/rubocop-hq/rspec-style-guide#should-in-example-docstrings), mainly to avoid repeating it a lot (similar comment for line 405)